saleor/graphql/meta/tests/mutations/test_shop.py

```diff
@@ -0,0 +1,46 @@
+
+import graphene
+
+from ....tests.utils import  get_graphql_content
+from . import PRIVATE_KEY, PRIVATE_VALUE, PUBLIC_KEY, PUBLIC_VALUE
+
+SHOP_SETTINGS_UPDATE_METADATA_MUTATION = """
+    mutation updateShopMetadata($input: ShopSettingsInput!) {
+        shopSettingsUpdate(input: $input) {
+            shop {
+            metadata {
+                key
+                value
+            }
+            privateMetadata {
+                key
+                value
+            }
+            }
+        }
+    }
+"""
```

Comment:
Thank you for moving the tests to the meta directory.

Could you right now make it consistent with other tests in `meta/tests/mutations`? For example take a look at this file: https://github.com/saleor/saleor/blob/main/saleor/graphql/meta/tests/mutations/test_gift_card.py. 

We need tests 4 tests, and please use the helper methods there:
- for updating public metadata (helper: `execute_update_public_metadata_for_item`)
- for updating private metadata (helper: `execute_clear_private_metadata_for_item`)
- for deleting public metadata (helper: `execute_clear_public_metadata_for_item`)
- for deleting private metadata (helper: `execute_clear_private_metadata_for_item`).